lib/rules/lint-attribute-indentation.js

```diff
@@ -360,6 +415,9 @@ module.exports = class AttributeSpacing extends Rule {
       if ('indentation' in config) {
         result.indentation = config.indentation;
       }
+      if ('process-elements' in config) {
+        result.parseHtmlElements = true;
```

Comment:
This seems odd, the way this is written it is impossible to disable `process-elements` explicitly.

I think that you can update this to `result.parseHtmlElements = config['process-elements']`, this way we will use `true` or `false` based on whatever the users configuration has in it.

Reply:
Makes sense, fixed and added a test in which I explicitly set `process-elements` to `false`.